tools/check-signed-off.sh

```diff
@@ -15,6 +15,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Usage
+function usage
+{
+ echo "$0: Check Signed-off-by message of latest commit"
+ echo ""
+ echo "Usage: $0 [--help] [--tolerant]"
+ echo ""
+ echo "Optional arguments:"
+ echo "  --help            print this help message"
+ echo "  --tolerant        check the existense of the message only but don't"
+ echo "                    require the name and email address to match the author"
+ echo "                    of the commit"
+ echo ""
+ echo "The last line of every commit message must follow the form of:"
+ echo "'JerryScript-DCO-1.0-Signed-off-by: NAME EMAIL', where NAME and EMAIL must"
+ echo "match the name and email address of the author of the commit (unless in"
+ echo "tolerant mode)."
+}
+
+# Processing command line
+if [ "$1" == "--help" ]
+then
+ usage
+ exit 0
+fi
+
+if [ "$1" == "--tolerant" ]
+then
+ TOLERANT="yes"
+ shift
+else
+ TOLERANT="no"
+fi
+
+if [ "$#" -gt 0 ]
```

Comment:
Print help and exit if there's at least one argument? I guess that should be "-gt 1" instead?  
My understanding is that the intended behavior is as follows:

- No argument: Check in strict mode
- --tolerant: Check in tolerant mode
- --help: Print help
- More than 1 argument: Print help and exit with error code  

The help should probably say something like "[--help | --tolerant]" to emphasize that an invocation with "--help --tolerant" will error out.